src/qt/editor/inviwoaboutwindow.cpp

```diff
@@ -186,6 +187,9 @@ InviwoAboutWindow::InviwoAboutWindow(InviwoMainWindow* mainwindow)
 
         auto uulm = p.append("a", "", {{"href", "http://www.uni-ulm.de/en/"}});
         uulm.append("img", "", makeImg(":/images/uulm.png", 50));
+
+        auto elliit = p.append("a", "", {{"href", "https://old.liu.se/elliit?l=en"}});
+        elliit.append("img", "", makeImg(":/images/elliit.png", 50));
```

Comment:
Have you tested this? I don't see any adding to the .rc file for the image?

Reply:
Ah, right. Added it and tested it. Changed to transparent background to be consistent with the others